arcgis-ios-sdk-samples/Augmented reality/Navigate in AR/NavigateARRoutePlannerViewController.swift

```diff
@@ -230,7 +231,6 @@ extension NavigateARRoutePlannerViewController: AGSGeoViewTouchDelegate {
 
 extension NavigateARRoutePlannerViewController: AGSAuthenticationManagerDelegate {
     func authenticationManager( _ authenticationManager: AGSAuthenticationManager, wantsToShow viewController: UIViewController) {
-        viewController.modalPresentationStyle = .overFullScreen
```

Comment:
Without this, how is the view controller presented on iPad? If I remember right, it presents fullscreen, which really does not look great.

Reply:
Right. There are 2 ways to fix this issue: 

1. Like the changes in this PR, we use the default fullScreen presentation style that doesn't allow dismiss by interactive swipe (dragging down)
2. Or we can set [isModalInPresentation](https://developer.apple.com/documentation/uikit/uiviewcontroller/3229894-ismodalinpresentation) to disable that in iOS 13+. But that will add a few other lines to check API availability. So I chose the first method even if it is a bit bad-looking.

Reply:
I actually think using `isModalInPresentation` on iOS 13+ is the way to go. It is more verbose, but I think it looks better (on screen) and shows users how they can do it in their own code.